lib/daemon/index.js

```diff
@@ -415,23 +460,20 @@ function removeSocketFiles() {
 
 exports.init = function () {
 	var processManager = require('../processManager');
+	var commandPid = getFilePid(CMD_LOCK_FILE);
 
-	// Disable stdout/stderr if we are starting daemonized
-	processManager.once('started', function () {
-		var commandPid = getFilePid(CMD_LOCK_FILE);
+	// Forward stdio to app process if daemonzied
+	setDaemonizedStdio(commandPid);
 
-		if (commandPid) {
-			const {
-				stdout,
-				stderr
-			} = process;
+	// Disable stdout/stderr if we are starting daemonized
+	processManager.once('started', () => disableDaemonizedStdio(commandPid));
 
-			stdout.write = () => false;
-			stderr.write = () => false;
-		}
+	// Upon exitting, check if we can forward exit logs
```

Comment:
Very nitpicky, but suggestion when it comes to comment writing. I try to avoid "check if", because usually there's something we do, if ...

Compare to: `// Upon exiting, try to forward exit logs`
(Also, `exiting` has 1 t)